precompiles/staking/staking.go

```diff
@@ -154,6 +156,7 @@ func (p Precompile) Run(evm *vm.EVM, contract *vm.Contract, readOnly bool) (bz [
 //
 // Available staking transactions are:
 //   - CreateValidator
+//   - EditValidator
 //   - Delegate
 //   - Undelegate
 //   - Redelegate
```

Comment:
I would get rid of this. It is the same list visible in the `switch method` but requires update and can be out of sync.